SIPs/template.md

```diff
@@ -0,0 +1,40 @@
+---
+sip: [Number of the SIP assigned by maintainer]
+title: [short title name]
+author: [name of the author]
+discussions-to: [link to the discussion, e.g., GitHub issue]
+status: [Draft | Review | Accepted | Rejected | Withdrawn | Implemented | Staging | Final]
+category: [Core | Networking | Interface | Library | Application | Informational]
+created: [YYYY-MM-DD]
+maintainer: [name of the maintainer, if different from author]
+---
+
+# SIP-X: [Short Title Here]
+
+## Abstract
+
+Short summary of the proposal.
+
+## Current
+
+Describe the current situation or behavior that needs improvement.
+
+## Suggested
```

Comment:
Same as above. For example: Suggested Changes, or even better, Proposed Changes.